stubs/D3DShot/d3dshot/capture_output.pyi

```diff
@@ -0,0 +1,36 @@
+import enum
+from _typeshed import Incomplete
+from ctypes import _CVoidConstPLike
+from typing_extensions import Literal, TypeAlias
+
+from PIL import Image
+
+_Frame: TypeAlias = Image.Image | Incomplete
+# _Frame: TypeAlias = Image.Image | npt.NDArray[np.int32] | npt.NDArray[np.float32] | _Tensor
+
+class CaptureOutputs(enum.Enum):
+    PIL: int
+    NUMPY: int
+    NUMPY_FLOAT: int
+    PYTORCH: int
+    PYTORCH_FLOAT: int
+    PYTORCH_GPU: int
+    PYTORCH_FLOAT_GPU: int
+
+class CaptureOutputError(BaseException): ...
+
+class CaptureOutput:
+    backend: CaptureOutput
```

Comment:
```suggestion
    backend: CaptureOutputs
```
didn't check the code but this would make more sense

Reply:
`CaptureOutput` is right. You pass the `CaptureOutputs` enum to `__init__` and it initializes a child class of `CaptureOutput` and stores it in `backend`.

Here's the source: https://github.com/SerpentAI/D3DShot/blob/dev/d3dshot/capture_output.py#L20

So `CaptureOutput` is both a base class and a wrapper.

Reply:
I agree with @Avasam's analysis, but adding a comment here might be good.